src/components/legend/attributes.js

```diff
@@ -105,6 +105,13 @@ module.exports = {
         editType: 'legend',
         description: 'Determines what entrywidth means.',
     },
+    indentation: {
+        valType: 'number',
+        min: 0,
```

Comment:
I could see us allowing negative indentation, to remove or reduce the padding to the left of many of the symbols. Many if not most legend symbols don't use the full `itemwidth`. The risk here is allowing symbols to bleed off the edge of the legend if you go too far, but I think that's acceptable if you have explicitly set a negative indentation. Maybe `min: -15`, ie half the minimum `itemwidth`?

Reply:
Yes the bleeding off the left edge was my concern as well. I have adapted the test image to showcase how it looks to indent with -15 for a line and a bar legend entry.

Reply:
Oh cool, we clip at the edge of the legend, that's perfect.